### PR TITLE
Added script to save during textual inversion training. Issue 524

### PR DESCRIPTION
**Pull request by isamu-isozaki**
Hi! I added a way to save during training

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
This looks good to me from a quick look! @patil-suraj what do you think?

File: examples/textual_inversion/textual_inversion.py [hunk 1]
```diff
@@ -29,8 +29,21 @@
...
+    print("Saving embeddings")
+    learned_embeds = accelerator.unwrap_model(text_encoder).get_input_embeddings().weight[placeholder_token_id]
+    learned_embeds_dict = {args.placeholder_token: learned_embeds.detach().cpu()}
+    torch.save(learned_embeds_dict, os.path.join(args.output_dir, "learned_embeds.bin"))
+
+
 def parse_args():
     parser = argparse.ArgumentParser(description="Simple example of a training script.")
+    parser.add_argument(
+        "--save_interval",
```
Comment (patil-suraj) [inline examples/textual_inversion/textual_inversion.py:42]: Let's call it `save_steps`
```suggestion
        "--save_steps",
```

File: examples/textual_inversion/textual_inversion.py [hunk 2]
```diff
@@ -29,8 +29,21 @@
...
+        type=int,
+        default=500,
+        help="Interval to save learned_embeds.bin",
```
Comment (patil-suraj) [inline examples/textual_inversion/textual_inversion.py:45]:
```suggestion
        help=""Save learned_embeds.bin every X updates steps.",
```

File: examples/textual_inversion/textual_inversion.py [hunk 3]
```diff
@@ -29,8 +29,21 @@
 logger = get_logger(__name__)
 
 
+def save_progress(text_encoder, placeholder_token_id, accelerator, args):
+    print("Saving embeddings")
```
Comment (patil-suraj) [inline examples/textual_inversion/textual_inversion.py:33]:
```suggestion
    logger.info("Saving embeddings")
```

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for the PR @isamu-isozaki, LGTM! Just left a couple of nits

File: examples/textual_inversion/textual_inversion.py (hunk 3 above)
Comment (isamu-isozaki) [reply examples/textual_inversion/textual_inversion.py:33]: Sounds good! Will fix now

**Comment by isamu-isozaki** [comment]
@patil-suraj Should be done!

**Comment by patil-suraj** [comment]
Thanks a lot!

**Pull request merged**